includes/admin/load.php

```diff
@@ -204,137 +207,142 @@ function perflab_dismiss_wp_pointer_wrapper() {
  * Callback function to handle admin scripts.
  *
  * @since 2.8.0
+ * @since n.e.x.t Renamed to perflab_enqueue_features_page_scripts().
  */
-function perflab_enqueue_modules_page_scripts() {
-	wp_enqueue_script( 'updates' );
-
-	wp_localize_script(
-		'updates',
-		'_wpUpdatesItemCounts',
-		array(
-			'settings' => array(
-				'totals' => wp_get_update_data(),
-			),
-		)
-	);
-
+function perflab_enqueue_features_page_scripts() {
 	wp_enqueue_script( 'thickbox' );
 	wp_enqueue_style( 'thickbox' );
 
 	wp_enqueue_script( 'plugin-install' );
```

Comment:
Maybe add a comment to explain why these are being enqueued? For example:
```suggestion
	// These assets are needed for the "Learn more" popover.
	wp_enqueue_script( 'thickbox' );
	wp_enqueue_style( 'thickbox' );
	wp_enqueue_script( 'plugin-install' );
```